### PR TITLE
Specification of quantized collective_broadcast op

### PR DESCRIPTION
**Pull request by sdasgup3**
Please find the specification of quantized `collective_broadcast` op. The specification is proposed to be similar to other distribution ops like `collective_permute`.

Let me know your feedback. 

@loganchien 
@paulinesho
@doyeonkim0

### CODE REVIEW

**Review by loganchien — APPROVED** [review]

**Review by doyeonkim0 — APPROVED** [review]

**Review by GleasonK — APPROVED** [review]

**Comment by GleasonK** [comment]
Wait for all relevant review approvals before merge, but LGTM.

**Review by abhigunj — APPROVED** [review]

**Pull request merged**
